sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -78,6 +79,26 @@ final class DataFrameWriter private[sql](df: DataFrame) {
   }
 
   /**
+   * Set the trigger interval for the stream query.
+   *
+   * @since 2.0.0
+   */
+  def trigger(interval: Duration): DataFrameWriter = {
```

Comment:
There is no Python `startStream`. I think we don't need to add a Python API that won't be used now.